### DISCUSSION TITLE
Integrations install error

### DISCUSSION THREAD

**Discussion by ozmel1** [mindsdb/mindsdb#9407] [Get help]
Hi, I have tried both gui and interactive shell to install integrations and get an error. the gui does not provide much details except to contact support but the interactive shell provides details. Here is one output of trying to pip install .[Snowflake].

Thank you for any help

C:\WINDOWS\system32>docker exec -it mindsdb_container sh
# pip install .[Snowflake]
Processing /mindsdb
  Installing build dependencies ... error
  error: subprocess-exited-with-error

  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Collecting setuptools
        WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1007)'))': /packages/b3/7a/629889a5d76200287aa5483d753811bd247bbd1b03175186f759e0c7d3a7/setuptools-70.1.1-py3-none-any.whl

**Comment by ZoranPandovski** [comment]
Hey @ozmel1 you need to run `pip install mindsdb[snowflake]`
